js/cart/cart-menu.js

```diff
@@ -0,0 +1,30 @@
+import { $ } from '../base/base.js';
+import { PRODUCT_STORAGE_KEY } from '../base/constant.js'
+import miniCart from '../component/cart-mini.js'
+
+const megamenuCart = $(".mega-menu--cart")
+
+const cartMenu = {
+    products: JSON.parse(localStorage.getItem(PRODUCT_STORAGE_KEY)) || [],
+
+    loadCart: function() {
+        let totalPrice = 0;
+        const cart = this.products.map((product, index) => {
+            totalPrice = totalPrice + product.sell
+            return miniCart(product);
+        })
+        const htmls = `${cart}
+            <div class="mega-menu__price">
+                <span class="mega-menu__total">Tổng số</span>
+                <span class="card__price">${totalPrice}đ</span></div>
+            <div class="button button-dark">Giỏ Hàng</div>
+        `
+        megamenuCart.innerHTML = htmls;
+    },
+
+    start: function () {
```

Comment:
```suggestion
    init: function () {
```
thông thường sẽ là như này chứ